odoo/addons/base/models/ir_binary.py

```diff
@@ -46,10 +46,10 @@ def _find_record(
         if not record:
             raise MissingError(f"No record found for xmlid={xmlid}, res_model={res_model}, id={res_id}")
 
-        record = self._find_record_check_access(record, access_token)
+        record = self._find_record_check_access(record, access_token, field)
         return record
 
-    def _find_record_check_access(self, record, access_token):
+    def _find_record_check_access(self, record, access_token, fields=None):
```

Comment:
not plural?

Also no need for a default on this method, it's always called with the param set